index.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Formula 1 Get Together Conference</title>
+    <link href='https://fonts.googleapis.com/css?family=Lato:400,700' rel='stylesheet' type='text/css'>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <header>
+        <div class="top-navbar">
+            <img src="images/facebook.svg" alt="" class="facebook-image">
+            <img src="images/twitter.svg" alt="" class="twitter-image">
+            <p>English</p>
+            <p>My Page</p>
+            <p>Logout</p>
+        </div>
+    </header>
+
+    <header class="hamburger-logo">
+    
+            <div class="header-container">
+                <a href="#"><img src="images/mobile_menu1.svg" alt="menu-image" class="ham"></a>
```

Comment:
Your design

![image](https://github.com/MarkYegon7/Project-Capstone/assets/78251927/e09a8aa7-6ca3-445f-86fe-6d68632142b2)

## Mobile
[OPTIONAL] - Kindly fix the issue with the hamburger menu so that it will be possible to access the `nav-links` and navigate between the pages 👍 🛠.